src/components/SearchFilter.vue

```diff
@@ -225,31 +240,32 @@ export default {
   computed: {
     ...mapState(['searchResultsPerPage', 'maxEntriesPerPage', 'uiLanguage']),
     ...mapGetters(['canSearchCollections', 'supportsConformance']),
+    collectionSearchLink() {
+      return this.parent instanceof CatalogLike && this.parent.getApiCollectionsLink();
+    },
     collectionSelectOptions() {
       let taggable = !this.hasAllCollections;
-      let isResult = this.collections.length > 0 && !this.hasAllCollections;
+      let isResult = this.collections.length > 0 && taggable;
       return {
         id: this.ids.collections,
-        value: this.selectedCollections,
+        options: this.collections,
         multiple: true,
-        taggable,
-        options: this.collections, // query.collections
-        trackBy: "value",
-        label: "text",
-        placeholder: taggable ? this.$t('search.enterCollections') : this.$t('search.selectCollections'),
+        taggable: taggable,
+        trackBy: 'value',
+        label: 'text',
+        placeholder: taggable 
+          ? this.$t('search.enterCollections') 
+          : this.$t('search.selectCollections'),
         tagPlaceholder: this.$t('search.addCollections'),
         selectLabel: this.$t('multiselect.selectLabel'),
         selectedLabel: this.$t('multiselect.selectedLabel'),
         deselectLabel: this.$t('multiselect.deselectLabel'),
-        limitText: count => this.$t("multiselect.andMore", {count}),
+        limitText: count => this.$t('multiselect.andMore', {count}),
         loading: this.collectionsLoadingTimer !== null,
         showNoResults: false,
-        internalSearch: !isResult
+        internalSearch: !isResult,
       };
     },
-    collectionSearchLink() {
-      return this.parent instanceof CatalogLike && this.parent.getApiCollectionsLink();
-    },
     canSearchCollectionsFreeText() {
```

Comment:
There's quite a bit of noise in this file diff. Can we revert to the original formatting as much as possible to ensure a clean history?